src/packages/media/media/components/input-image-cropper/image-cropper-focus-setter.element.ts

```diff
@@ -9,36 +12,48 @@ export class UmbImageCropperFocusSetterElement extends LitElement {
 	imageElement!: HTMLImageElement;
 
 	@query('#wrapper')
-	wrapperElement?: HTMLImageElement;
+	wrapperElement?: HTMLElement;
 
 	@query('#focal-point')
-	focalPointElement!: HTMLImageElement;
+	focalPointElement!: HTMLElement;
+
+	@state() private isDraggingGridHandle = false;
```

Comment:
```suggestion
	@state()
	private isDraggingGridHandle = false;
```

Reply:
Should also be renamed to `_isDraggingGridHandle` so it's ignored in the debug element.